docs/tools/vdb_table/data/pinecone.json

```diff
@@ -29,13 +29,13 @@
   "vector_launch_year": 2019,
   "metadata_filter": {
     "support": "full",
-    "source_url": "",
+    "source_url": "https://docs.pinecone.io/docs/metadata-filtering",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "none",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
```

Comment:
I read through the example: https://github.com/pinecone-io/examples/blob/master/learn/search/hybrid-search/ecommerce-search/ecommerce-search.ipynb linked from that page, and it seems odd to me that the actual vectors are being scaled by the alpha amount to do a linear combination of the lists from sparse and dense search.

Shouldn't the results be the same regardless of the scale of the vector, given that the nearest neighbor metric could be something like cosine distance?

Reply:
it uses dot product (not cosine), so we can scale the impact from sparse vs. dense vectors :)

When using hybrid you need to use dotproduct rather than cosine or euclidean